src/NTDll/NTDll+PROCESS_BASIC_INFORMATION.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="PROCESS_BASIC_INFORMATION"/> nested type.
+    /// </content>
+    public static partial class NTDll
+    {
+        /// <summary>
+        /// Contains basic information about a process.
+        /// </summary>
+        /// <seealso href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms684280(v=vs.85).aspx"/>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct PROCESS_BASIC_INFORMATION
+        {
+            /// <summary>
+            /// This value is reserved.
+            /// </summary>
+            public IntPtr Reserved1;
+
+            /// <summary>
+            /// The base address of the PEB structure in the process memory.
+            /// </summary>
+            public IntPtr PebBaseAddress;
+
+            /// <summary>
+            /// This value is reserved.
+            /// </summary>
+            public IntPtr Reserved2a;
+
+            /// <summary>
+            /// This value is reserved.
+            /// </summary>
+            public IntPtr Reserved2b;
```

Comment:
This is an interesting one. It's a fixed size array in a struct. C# does actually have a way to represent this so you don't have to invent two unique field names. See [this example](https://github.com/dotnet/pinvoke/blob/b0cb7c1698d92193c58ce32209de5236a4f6bc9d/src/Kernel32/storebanned/Kernel32%2BPROCESSENTRY32.cs#L81). Can you used a fixed size `IntPtr` array here too?

Reply:
I don't think you can use `IntPtr` in fixed size arrays, though:

> Fixed size buffer type must be one of the following: bool, byte, short, int, long, char, sbyte, ushort, uint, ulong, float or double

Reply:
Gah! OK then. As you were.